HOMEWORK.js

```diff
@@ -58,31 +58,36 @@ function homework (renderList) {
 	// ◽ super bonus: remove the X if it already has one (put it back to incomplete)
 	function onCompleteTodo (index) {
 		// Add code here
-
+		todos[index].charAt(0) === 'X' ? todos[index] = todos[index].slice(1) : todos[index] = 'X' + todos[index];
```

Comment:
See previous comment about ternary versus if/else statements.

It is possible to write this line so it looks more like the normal ternary situation
```js
thingToUpdate = conditional ? trueValue : falseValue
```
It might be clearer if you can rewrite it in this form. Try it out!